operator/src/main/resources/scripts/startServer.sh

```diff
@@ -102,8 +102,12 @@ function startWLS() {
 
   FAIL_BOOT_ON_SITUATIONAL_CONFIG_ERROR=${FAIL_BOOT_ON_SITUATIONAL_CONFIG_ERROR:-true}
   SERVER_OUT_MONITOR_INTERVAL=${SERVER_OUT_MONITOR_INTERVAL:-3}
-  if [ ${FAIL_BOOT_ON_SITUATIONAL_CONFIG_ERROR} == 'true' ] ; then
-    ${SCRIPTPATH}/monitorLog.sh ${SERVER_OUT_FILE} ${SERVER_OUT_MONITOR_INTERVAL} &
+  if [ ! -f /weblogic-operator/introspector/domainzip.secure ]; then
```

Comment:
This check looks a little obscure/fragile to me. Instead, how about checking the DOMAIN_SOURCE_TYPE env var? (You'll need to make sure that this env var is passed to all servers - the operator might only be passing it to the introspector pod right now...)